double_entry.gemspec

```diff
@@ -17,6 +17,31 @@ Gem::Specification.new do |gem|
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.require_paths = ['lib']
 
+  gem.post_install_message = <<-'POSTINSTALLMESSAGE'
+Please note the following changes in DoubleEntry version 10.4:
```

Comment:
Aren't we on version 0.10.3?

Also, I was thinking that the metadata changes might graduate DoubleEntry to 1.0.0 :) (so yes)